ntcore/src/generate/main/java/Topic.java.jinja

```diff
@@ -222,5 +222,11 @@ public final class {{ TypeName }}Topic extends Topic {
         defaultValue);
   }
 {% endif %}
+
+  @Override
+  public void close() {
+    m_inst.flush();
+    m_inst.close();
```

Comment:
This is not at all what this should do.  Closing just one topic should not close the entire instance.